mobile/src/main/java/org/openhab/habdroid/ui/WidgetAdapter.kt

```diff
@@ -114,6 +114,7 @@ class WidgetAdapter(
     private val inflater = LayoutInflater.from(context)
     private val chartTheme: CharSequence
     private var selectedPosition = -1
+    private var firstVisibleWidgetPosition = -1
```

Comment:
Maybe use a constant here:

```suggestion
    private var selectedPosition = RecyclerView.NO_POSITION
    private var firstVisibleWidgetPosition = RecyclerView.NO_POSITION
```

Reply:
@maniac103 What do you think about my suggestion?

Reply:
It's fine ... I just forgot to update the branch :-/ I'll update it tomorrow.